packages/modules/src/modules/banners/momentTemplate/components/MomentTemplateBannerArticleCountOptOut.tsx

```diff
@@ -0,0 +1,225 @@
+import React, { useState } from 'react';
```

Comment:
This file is essentially a simplified version of [this one](https://github.com/guardian/support-dotcom-components/blob/main/packages/modules/src/modules/shared/ArticleCountOptOutPopup.tsx). 

It instead takes a `BannerTemplateSettings` object to specify the settings and I've also reordered things to hopefully make it a little cleaner/readable